src/patternfly/components/Tabs/tabs.scss

```diff
@@ -112,167 +126,305 @@
   max-width: 100%;
   overflow-x: auto;
   -webkit-overflow-scrolling: touch;
-
-  // Right border on last tab
-  .pf-c-tabs__item:last-of-type .pf-c-tabs__button::before {
-    border-right-width: var(--pf-c-tabs__item--BorderWidth);
-  }
 }
 
-// Tab items
+// Tabs item
 .pf-c-tabs__item {
+  display: flex;
   flex: none;
+  align-items: stretch;
+
+  // Current
+  &.pf-m-current {
+    .pf-c-tabs__button,
+    .pf-c-tabs__button:hover::after {
+      --pf-c-tabs__button--Color: var(--pf-c-tabs__item--m-current--c-tabs__button--Color);
+      --pf-c-tabs__button--after--BorderColor: var(--pf-c-tabs__item--m-current--c-tabs__button--after--BorderColor);
+    }
+  }
```

Comment:
could this block be written as

```
  // Current
  &.pf-m-current {
    --pf-c-tabs__button--Color: var(--pf-c-tabs__item--m-current--c-tabs__button--Color);
    --pf-c-tabs__button--after--BorderColor: var(--pf-c-tabs__item--m-current--c-tabs__button--after--BorderColor);
    --pf-c-tabs__button--hover--after--BorderColor: var(--pf-c-tabs__item--m-current--c-tabs__button--after--BorderColor);
  }
```

Reply:
`var` need to be set on  `.pf-c-tabs__button` / has no effect if set on parent